src/Command/Dispatcher/DispatchFilesCommand.php

```diff
@@ -295,6 +295,28 @@ private function deleteNotNeededFilesAndDirs(Project $project, Branch $branch, s
             }
         }
 
+        if (!$branch->hasFrontend()) {
+            $filepaths = [
+                '.babelrc.js',
+                '.eslintrc.js',
+                '.stylelintrc.js',
+                'postcss.config.js',
+                '.github/workflows/frontend.yaml',
+            ];
+
+            foreach ($filepaths as $filepath) {
```

Comment:
Shall we extract a method to remove a file by path?

Reply:
I think it should be better now